zoltar-core/src/test/java/com/spotify/zoltar/PredictorTest.java

```diff
@@ -70,7 +70,7 @@ public void timeout() {
           .get(wait.toMillis(), TimeUnit.MILLISECONDS);
 
       fail("should throw TimeoutException");
-    } catch (Exception e) {
+    } catch (final Exception e) {
```

Comment:
Is there a maven plugin that could do this for us?

Reply:
@ravwojdyla hmm not sure! have to investigate 😄